### PR TITLE
[RO-396] Added updatedAt property to LineItem entity

### PR DESCRIPTION
**Pull request by Skyld**
Ticket: https://oat-sa.atlassian.net/browse/RO-396

### Added
- Added `updated_at` field to `line_items` database table.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -19,8 +19,7 @@ JWT_REFRESH_TOKEN_TTL=86400
 # IMPORTANT: You MUST configure your server version, either here or in config/packages/doctrine.yaml
 #
 # DATABASE_URL="sqlite:///%kernel.project_dir%/var/data.db"
-# DATABASE_URL="mysql://db_user:db_password@127.0.0.1:3306/db_name?serverVersion=13&charset=utf8"
-# DATABASE_URL="postgresql://db_user:db_password@127.0.0.1:5432/db_name?serverVersion=13&charset=utf8"
```
Comment (uncleempty) [inline .env:23]: why postgresql example is gone?

File: config/packages/framework.yaml [hunk 2]
```diff
@@ -1,16 +1,24 @@
 # see https://symfony.com/doc/current/reference/configuration/framework.html
 framework:
     secret: '%env(APP_SECRET)%'
-    default_locale: '%locale%'
```
Comment (uncleempty) [inline config/packages/framework.yaml:4]: why default_locale is gone?

File: config/packages/framework.yaml [hunk 3]
```diff
@@ -1,16 +1,24 @@
...
+    http_method_override: false
 
     # Enables session support. Note that the session will ONLY be started if you read or write from it.
     # Remove or comment this section to explicitly disable session support.
     session:
-        enabled: false
+        handler_id: null
+        cookie_secure: auto
+        cookie_samesite: lax
+        storage_factory_id: session.storage.factory.native
```
Comment (uncleempty) [inline config/packages/framework.yaml:13]: why this is back? This was gone somewhere around switching to JWT. Are you sure that it won't enable sessions back?

File: config/packages/prod/deprecations.yaml [hunk 4]
```diff
@@ -5,4 +5,4 @@
 #        deprecation:
 #            type: stream
 #            channels: [deprecation]
-#            path: "%kernel.logs_dir%/%kernel.environment%.deprecations.log"
+#            path: php://stderr
```
Comment (uncleempty) [inline config/packages/prod/deprecations.yaml:8]: Also - why? Previously we had a kind of a good "source of truth" to watch for the deprecations. How will we do that now?

**Review by uncleempty — COMMENTED** [review]
Good one, thanks! Though I have some questions

File: config/packages/framework.yaml (hunk 2 above)
Comment (Skyld) [reply config/packages/framework.yaml:4]: Flex recipe... reverted

File: .env (hunk 1 above)
Comment (Skyld) [reply .env]: Flex recipe... reverted

File: config/packages/framework.yaml (hunk 3 above)
Comment (Skyld) [reply config/packages/framework.yaml]: Flex recipe... reverted

File: config/packages/prod/deprecations.yaml (hunk 4 above)
Comment (Skyld) [reply config/packages/prod/deprecations.yaml]: Flex recipe... reverted

**Review by uncleempty — APPROVED** [review]
Good job Dani

**Review Checklist**
- [X] New code is covered by tests (if applicable)
- [X] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [X] New code is respecting code style rules
- [X] New code is respecting best practices
- [X] New code is not subject to concurrency issues (if applicable)
- [X] Feature is working correctly on my local machine (if applicable)
- [X] Acceptance criteria are respected
- [X] Pull request title and description are meaningful
- [X] Changelog is updated according to changes (if applicable)
- [X] Documentation is updated according to changes (if applicable)

**Review by luisfn — APPROVED** [review]
Thanks for the changes Dani

**Review Checklist**
- [X] New code is covered by tests (if applicable)
- [X] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [X] New code is respecting code style rules
- [X] New code is respecting best practices
- [X] New code is not subject to concurrency issues (if applicable)
- [X] Feature is working correctly on my local machine (if applicable)
- [X] Acceptance criteria are respected
- [X] Pull request title and description are meaningful
- [X] Changelog is updated according to changes (if applicable)
- [X] Documentation is updated according to changes (if applicable)

**Pull request merged**